Update interfaces/webrtc.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/5560203/ed/idl/webrtc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/590400095

lukebjerring commented on interfaces/webrtc.idl:
```diff
@@ -121,25 +122,33 @@ interface RTCPeerConnection : EventTarget {
                                 RTCPeerConnectionErrorCallback failureCallback);
 };
 
+          tests: 1
```
Something went wrong...

foolip commented:
Filed https://github.com/tidoust/reffy/issues/181.

autofoolip committed:
Update interfaces/webrtc.idl

Source: https://github.com/tidoust/reffy-reports/blob/5560203/ed/idl/webrtc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/590400095

foolip reviewed (approved):
Reffy bug fixed. This now makes the validation subtest pass. Yay!
